### ISSUE TITLE
Tighter integration with Netlify

### ISSUE THREAD

**Issue by austincondiff** [netlify/netlify-cms#3352]
It may be useful for users to be able to easily switch between the admins of various Netlify/Netlify CMS websites from any site that given user manages or to be able to access form data, analytics, and deploys, that typically would be in the Netlify app directly from Netlify CMS. 

I am not suggesting that we rebuild the Netlify app inside of Netlify CMS, however I do think it would be beneficial to include some information where it makes sense.

**Comment by martinjagodic** [comment]
@austincondiff I agree, but since we want to be platform agnostic, I updated the issue title to reflect this.

Easiest to implement:
- add a link to hosting (eg. Netlify site)
- show build status

other ideas:
- show latest per-branch deploys with preview links
- show latest edits (sort of git blame)

I would not include form submissions, these could show private information to the wrong person.

Statistics are behind a paywall, and we would also have to make some graphs which takes some work.
